From 317cae72408a06af09acb74dfc3c3554be87ed29 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Mon, 16 Feb 2015 15:58:32 -0500
Subject: [PATCH 071/315] session: allow display to be set after object
 construction

When we support starting the X server in the session, we're
not going to know the display name up front.

This commit makes it possible to set the display name
later.

https://bugzilla.gnome.org/show_bug.cgi?id=744764
---
 daemon/gdm-session.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/gdm-session.c b/daemon/gdm-session.c
index 4c2a8db1..55f67096 100644
--- a/daemon/gdm-session.c
+++ b/daemon/gdm-session.c
@@ -3497,7 +3497,7 @@ gdm_session_class_init (GdmSessionClass *session_class)
                                                               "display name",
                                                               "display name",
                                                               NULL,
-                                                              G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY));
+                                                              G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_DISPLAY_HOSTNAME,
                                          g_param_spec_string ("display-hostname",
-- 
2.15.1

